tests/components/test_pipeline_component.py

```diff
@@ -1,85 +1 @@
-from kpops.cli.pipeline_config import TopicNameConfig
-from kpops.component_handlers import ComponentHandlers
-from kpops.components.base_components.models.to_section import (
-    OutputTopicTypes,
-    TopicConfig,
-    ToSection,
-)
-from kpops.components.base_components.pipeline_component import PipelineComponent
-from kpops.utils.environment import ENV
-
-DEFAULTS_PATH = Path(__file__).parent / "resources"
-
-
-class PlainPipelineComponent(PipelineComponent):
-    type: str = "plain-pipeline-component"
-
-
-@pytest.fixture
-def prefix_to_match() -> Generator[str, None, None]:
-    prefix = "my-fake-prefix"
-    ENV["pipeline_name"] = prefix
-    yield prefix
-    del ENV["pipeline_name"]
-
-
-class TestPipelineComponent:
-    def test_topic_substitution(self):
-        pipeline_component = PlainPipelineComponent(
-            name="test-pipeline-component",
-            config=pipeline_config.PipelineConfig(
-                defaults_path=DEFAULTS_PATH,
-                environment="development",
-                topic_name_config=TopicNameConfig(
-                    default_error_topic_name="error-${component_type}",
-                    default_output_topic_name="output-${component_type}",
-                ),
-            ),
-            handlers=ComponentHandlers(
-                schema_handler=MagicMock(),
-                connector_handler=MagicMock(),
-                topic_handler=MagicMock(),
-            ),
-            to=ToSection(
-                models={},
-                topics={
-                    "${error_topic_name}": TopicConfig(
-                        type=OutputTopicTypes.ERROR, partitions_count=10
-                    ),
-                    "${output_topic_name}": TopicConfig(
-                        type=OutputTopicTypes.OUTPUT, partitions_count=10
-                    ),
-                },
-            ),
-        )
-
-        assert pipeline_component.to
-        assert "error-plain-pipeline-component" in pipeline_component.to.topics
-        assert "output-plain-pipeline-component" in pipeline_component.to.topics
-        assert len(pipeline_component.to.topics.keys()) == 2
-
-    def test_prefix_substitution(self, prefix_to_match: str):
-        pipeline_component = PlainPipelineComponent(
-            name="test-pipeline-component",
-            config=pipeline_config.PipelineConfig(
-                defaults_path=DEFAULTS_PATH,
-                environment="development",
-                topic_name_config=TopicNameConfig(
-                    default_error_topic_name="error-${component_type}",
-                    default_output_topic_name="output-${component_type}",
-                ),
-            ),
-            handlers=ComponentHandlers(
-                schema_handler=MagicMock(),
-                connector_handler=MagicMock(),
-                topic_handler=MagicMock(),
-            ),
-        )
-        assert prefix_to_match + "-" == pipeline_component.prefix
+pass
```

Comment:
what happened here?

Reply:
see:
[`56b46dd` (#198)](https://github.com/bakdata/kpops/pull/198/commits/56b46ddd583ad06978a26efab89564e6e3fe7d17)
[`006f16a` (#198)](https://github.com/bakdata/kpops/pull/198/commits/006f16a899ef7bea5d3b6956c7d26b8b7bf281b4)
https://github.com/bakdata/kpops/pull/198#discussion_r1202474325

I moved the tests, should I delete the file?

Reply:
yeah, please delete it

Reply:
done